xkcdgenerator/Makefile

```diff
@@ -0,0 +1,21 @@
+# Makefile for xkcdgenerator
+
+PROJECT  = xkcdgenerator
+VERSION  = $(shell cargo metadata --no-deps --format-version 1 | jq -r '.packages[] .version' | head -1)
+REVISION = 0
+# list of all contract claims for actor signing (space-separated)
+CLAIMS   = wasmcloud:httpserver wasmcloud:httpclient wasmcloud:builtin:numbergen
+# registry url for our actor
+REG_URL  = localhost:5000/v2/$(PROJECT):$(VERSION)
```

Comment:
I'm torn on whether we should continue to propagate this gross `/v2` part of the URI...

Reply:
Yeah I just left this from the template :/ we should probably address this in the template